gandi/cli/commands/vm.py

```diff
@@ -28,8 +28,10 @@ def vm(gandi):
 @click.option('--id', help='Display ids.', is_flag=True)
 @click.option('--limit', help='Limit number of results.', default=100,
               show_default=True)
+@click.option('--format', type=click.Choice(['json', 'pretty-json']),
+              required=False, help="Choose the output format")
 @pass_gandi
-def list(gandi, state, id, limit, datacenter):
+def list(gandi, state, id, limit, datacenter, format):
```

Comment:
`format` seems to expect a boolean, `True` for JSON and `False` for the old output format.

For me the argument name is confusing, and makes me assume that some kind of a value signifying the output format (e.g. the string `"json"`) would be expected.

Wouldn't e.g. `json_format`, or just `json` be a better argument name? Or, alternatively, accepting `"vm"` or `"json"` as values instead of booleans?